ember-toucan-core/src/components/form/radio-group-field.ts

```diff
@@ -15,11 +15,7 @@ export interface ToucanFormRadioGroupFieldComponentSignature {
     label: string;
     name: string;
     onChange?: ToucanFormRadioFieldComponentSignature['Args']['onChange'];
-    /**
-     * A test selector for targeting the root element of the field. In this case, the wrapping div element.
-     */
-    rootTestSelector?: string;
-    value?: ToucanFormRadioFieldComponentSignature['Args']['value'];
+    value: ToucanFormRadioFieldComponentSignature['Args']['value'];
```

Comment:
I also tried adding an assert like:

```ts
  constructor(
    owner: unknown,
    args: ToucanFormRadioGroupFieldComponentSignature['Args']
  ) {
    assert('A "@value" argument is required', value);
    super(owner, args);
  }
```

but was getting errors in the docs-app on render/mount.  I'm thinking that due to the way yielded components work or something that `@value` isn't supplied or something on first render, but then shortly thereafter? I have 0 evidence this is the case but wanted to point out that I tried this and it didn't work 😭

Reply:
Did you try adding `this.args` before `value` in the assert?

Like this:
```
assert('A "@value" argument is required', this.args.value);
```

Reply:
... I am so dumb... I forgot to update the UI States section so _those_ were throwing errors and causing this to fail. Nothing to see here! 🙈